ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/commands.proto

```diff
@@ -47,12 +47,10 @@ message Commands {
   // Required
   string party = 5;
 
-  // MUST be an approximation of the wall clock time on the ledger server.
-  // Required
+  // Deprecated. This field currently has the same function as min_ledger_time_abs, and will be removed in the future.
   google.protobuf.Timestamp ledger_effective_time = 6;
```

Comment:
You should change this to `reserved` then so nobody can use this field (it will be just ignored when specified by a client):
https://developers.google.com/protocol-buffers/docs/proto#reserved

Reply:
Done in a follow up pr: #5172

Reply:
Great!